PrimeNumber.java

```diff
@@ -12,6 +12,8 @@ public static void main (String[] args)
 		boolean c = isPrime(n);
 		System.out.println(n + "is a Prime Number :" + c );
 	 
+	 
+	 
 	}
 	
 	public static boolean isPrime(int n)
```

Comment:
Why don't you add runtime complexity information here ? I will see if it's correct and then approve the changes!